graylog2-web-interface/src/components/common/InputDescription.jsx

```diff
@@ -0,0 +1,56 @@
+// @flow strict
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import styled, { type StyledComponent } from 'styled-components';
+
+import type { ThemeInterface } from 'theme';
+import { HelpBlock } from 'components/graylog';
+
+const Wrapper: StyledComponent<{hasError: boolean}, ThemeInterface, HTMLSpanElement> = styled.span(({ theme, hasError }) => `
```

Comment:
I find that putting a conditional with a template string and using a template string again can be confusing. Personally I think this reads better:

```suggestion
const Wrapper: StyledComponent<{hasError: boolean}, ThemeInterface, HTMLSpanElement> = styled.span(({ theme, hasError }) => hasError ? `color: ${theme.colors.variant.danger};` : '';
```

Reply:
Thanks, thats a good hint!